webpack.config.js

```diff
@@ -41,6 +42,17 @@ const shouldCheckForCycles = process.env.CHECK_CYCLES === 'true';
 const codeSplit = config.isEnabled( 'code-splitting' );
 const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 
+/**
+ * Plugin that generates the `public/custom-properties.css` file before compilation
+ */
+class BuildCustomPropertiesCssPlugin {
+	apply( compiler ) {
+		compiler.hooks.compile.tap( 'BuildCustomPropertiesCssPlugin', () =>
+			execSync( 'node bin/build-custom-properties-css.js' )
```

Comment:
are we guaranteed on the current working dir here with the sdk?

Reply:
> are we guaranteed on the current working dir here with the sdk?

Not sure. I don't even know what scenarios to test for that.

Reply:
I just experimented a bit... and building from a different cwd doesn't work even on `master`. Topic for another PR, as it's a frequent request to make it work.